e2e/single-cluster/single_cluster_test.go

```diff
@@ -96,5 +97,48 @@ var _ = Describe("Single Cluster Deployments", func() {
 				}).Should(ContainSubstring("mp-app-service"))
 			})
 		})
+		// This test fails from v0.9.1-rc.2 to v0.9.1-rc.6 due to issue:
+		// https://github.com/rancher/fleet/issues/2128
+		Context("containing multiple deployments", func() {
+			BeforeEach(func() {
+				asset = "single-cluster/helm-status-check.yaml"
+			})
+
+			It("all deployments are ready and status shown is Ready", func() {
+				Eventually(func() bool {
+					frontendReady := checkNReplicasAreReady(k, "frontend", "2")
```

Comment:
As discussed: Ideally we would be able to use less heavy deployments with smaller image sizes and no dependencies on physical volumes, etc.

Reply:
Yup, moving this to draft so it's not merged by mistake

Reply:
I've changed the test and now it's a simple `k8s.gcr.io/pause` container which should be less than 250kb and just by adding a init container sleeping 5 seconds I was able to recreate the issue with the affected RCs.